module/spl/spl-kmem.c

```diff
@@ -133,6 +133,79 @@ strfree(char *str)
 }
 EXPORT_SYMBOL(strfree);
 
+/*
+ * Try hard to satisfy the allocation. However, when progress
+ * cannot be made, the allocation is allowed to fail.
+ */
+#ifndef __GFP_RETRY_MAYFAIL
+#define	__GFP_RETRY_MAYFAIL	__GFP_REPEAT
+#endif
+
+/*
+ * This is basically a backport oflinux' kvmalloc_node implementation
+ * which is available as of v4.12.
+ */
```

Comment:
Let's drop this comment.  The proposed updated version of `spl_kvmalloc()` unconditionally calls `kvmalloc_node` for 4.13 and newer kernels so this should be self evident.